client/src/components/profileInfoEdit/ProfileInfoEdit.js

```diff
@@ -21,22 +21,49 @@ const ProfileInfoEdit = () => {
   const dispatch = useDispatch();
 
   const {
-    teamId, teamName, orgName, email,
+    teamId, teamName, orgName, email, profilePic,
```

Comment:
You may want to define `profilePic` in the reducer as well: https://github.com/Researchify/Researchify/blob/237b45c4d74421cf45cb1517775fa305f5b9a0a9/client/src/reducers/teamReducer.js#L15-L24

unless you plan to integrate with Redux later?